N/NetCDF/build_tarballs.jl

```diff
@@ -55,7 +55,7 @@ if [[ ${target} -ne x86_64-linux-gnu ]]; then
 fi
 
 # https://github.com/JuliaPackaging/Yggdrasil/issues/5031#issuecomment-1155000045
-rm /workspace/destdir/lib/*.la
+rm -f /workspace/destdir/lib/*.la
```

Comment:
If you need `-f` because there are no libtool files anymore (noways we automatically remove them) then you may as well just remove this.

Reply:
ok, thanks, I just updated the PR